armi/reactor/blocks.py

```diff
@@ -1570,6 +1570,7 @@ def _createHomogenizedCopy(self, pinSpatialLocators=False):
 
         b = self.__class__(self.getName(), height=self.getHeight())
         b.setType(self.getType())
+        b.p.flags = self.p.flags
```

Comment:
I think setting the flags from within the `setType()` call itself would be more efficient, because it would avoid the internal call to `Flags.fromStringIgnoreErrors(typ)`.

https://github.com/terrapower/armi/blob/main/armi/reactor/composites.py#L757-L784

Reply:
But either should work. I'm testing the way that you have it right now.

Reply:
Updated!